fuse/ipns/ipns_test.go

```diff
@@ -80,17 +129,29 @@ func setupIpnsTest(t *testing.T, node *core.IpfsNode) (*core.IpfsNode, *fstest.M
 	return node, mnt
 }
 
+func TestIpnsLocalLink(t *testing.T) {
+	_, mnt := setupIpnsTest(t, nil)
+	defer mnt.Close()
+	name := mnt.Dir + "/local"
+
+	finfo, err := os.Stat(name)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
yeah, I can check some more stuff. This test was originally just a quick verification that fuse was doing its thing,